Connecting SPI3 in DeviceTree to the board based on stm32f413xx generates FATAL ERROR while compiling

**Describe the bug**
For my project was created a custom BOARD based on stm32f413vg microcontroller. When I try to add SPI3 to the DeviceTree file it generate FATAL ERROR while compiling.  SPI1, SPI2, SPI4, SPI5, are have not that problem.
I tried the same with BOARD nucleo_f413zh, and got the same situation: SPI3 generates the ERROR.

**To Reproduce**
For Reproduce this ERROR needs to use project from the Zephyr samples "blinky".
1. To the project zephyr/samples/basic/blinky/boards/ need to add nucleo_f413zh.overlay which contains next content:
`&spi3 {
	pinctrl-0 = <&spi3_sck_pc10 &spi3_miso_pc11 &spi3_mosi_pc12>;
	pinctrl-names = "default";
	cs-gpios = <&gpioa 15 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
	status = "okay";
};`

2. CMakeLists.txt contains next content:
`cmake_minimum_required(VERSION 3.20.0)
set (DTC_OVERLAY_FILE "boards/nucleo_f413zh.overlay")
find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
project(blinky)
target_sources(app PRIVATE src/main.c)`

3. To compile the project:
`cd zephyr/samples/basic/blinky
west build -b nucleo_f413zh`

**Logs and console output**
The output of the errors we get:
`-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/samples/basic/blinky
-- CMake version: 3.22.1
-- Found Python3: /home/sintez/work/orion-sw-periphery-zephyr-workspace/.venv/bin/python3.10 (found suitable exact version "3.10.6") found components: Interpreter 
-- Cache files will be written to: /home/sintez/.cache/zephyr
-- Zephyr version: 3.3.0-rc1 (/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr)
-- Found west (found suitable version "0.14.0", minimum required is "0.7.1")
-- Board: nucleo_f413zh
-- ZEPHYR_TOOLCHAIN_VARIANT not set, trying to locate Zephyr SDK
-- Found host-tools: zephyr 0.15.0 (/home/sintez/zephyr-sdk-0.15.0)
-- Found toolchain: zephyr 0.15.0 (/home/sintez/zephyr-sdk-0.15.0)
-- Found Dtc: /home/sintez/zephyr-sdk-0.15.0/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/boards/arm/nucleo_f413zh/nucleo_f413zh.dts
-- Found devicetree overlay: boards/nucleo_f413zh.overlay
devicetree error: ./boards/nucleo_f413zh.overlay:18 (column 1): parse error: undefined node label 'spi3'
-- In: /home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/samples/basic/blinky/build/zephyr, command: /home/sintez/work/orion-sw-periphery-zephyr-workspace/.venv/bin/python3.10;/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/scripts/dts/gen_defines.py;--dts;/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/samples/basic/blinky/build/zephyr/zephyr.dts.pre;--dtc-flags;'';--bindings-dirs;/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/dts/bindings;--header-out;/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/samples/basic/blinky/build/zephyr/include/generated/devicetree_generated.h.new;--dts-out;/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/samples/basic/blinky/build/zephyr/zephyr.dts.new;--edt-pickle-out;/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/samples/basic/blinky/build/zephyr/edt.pickle;--vendor-prefixes;/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/dts/bindings/vendor-prefixes.txt
CMake Error at /home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/cmake/modules/dts.cmake:231 (message):
  gen_defines.py failed with return code: 1
Call Stack (most recent call first):
  /home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/cmake/modules/zephyr_default.cmake:113 (include)
  /home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include)
  /home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:92 (include_boilerplate)
  CMakeLists.txt:5 (find_package)


-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: /usr/bin/cmake -DWEST_PYTHON=/home/sintez/work/orion-sw-periphery-zephyr-workspace/.venv/bin/python3 -B/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/samples/basic/blinky/build -GNinja -DBOARD=nucleo_f413zh -S/home/sintez/work/orion-sw-periphery-zephyr-workspace/zephyr/samples/basic/blinky`

**Environment (please complete the following information):**
- OS: Ubuntu 22.04.1 LTS
- Toolchain: Zephyr SDK / GCC
- Zephyr version: 3.3.0

--- a/dts/arm/st/f4/stm32f412.dtsi
+++ b/dts/arm/st/f4/stm32f412.dtsi
@@ -50,6 +50,16 @@
 			status = "disabled";
 		};
 
+		spi3: spi@40003c00 {
+			compatible = "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40003c00 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00008000>;
+			interrupts = <51 5>;
+			status = "disabled";
+		};
+
 		spi4: spi@40013400 {
 			compatible = "st,stm32-spi";
 			#address-cells = <1>;

